doc/rbd/rbd-snapshot.rst

```diff
@@ -66,75 +73,94 @@ Create Snapshot
 ---------------
 
 To create a snapshot with ``rbd``, specify the ``snap create`` option, the pool
-name and the image name.  ::
+name, and the image name:
 
-	rbd snap create {pool-name}/{image-name}@{snap-name}
+.. prompt:: bash $
 
-For example:: 
+   rbd snap create {pool-name}/{image-name}@{snap-name}
 
-	rbd snap create rbd/foo@snapname
+For example:
+
+.. prompt:: bash $
+
+   rbd snap create rbd/foo@snapname
 	
 
 List Snapshots
 --------------
 
-To list snapshots of an image, specify the pool name and the image name. ::
+To list the snapshots of an image, specify the pool name and the image name:
 
-	rbd snap ls {pool-name}/{image-name}
+.. prompt:: bash $
 
-For example::
+   rbd snap ls {pool-name}/{image-name}
 
-	rbd snap ls rbd/foo
+For example:
 
+.. prompt:: bash $
 
-Rollback Snapshot
------------------
+   rbd snap ls rbd/foo
 
-To rollback to a snapshot with ``rbd``, specify the ``snap rollback`` option, the
-pool name, the image name and the snap name. ::
 
-	rbd snap rollback {pool-name}/{image-name}@{snap-name}
+Roll back Snapshot
+------------------
 
-For example::
+To roll back to a snapshot with ``rbd``, specify the ``snap rollback`` option,
```

Comment:
Ditto -- "To roll back an image to a snapshot, use `rbd snap rollback` command and specify ...".

Reply:
Done.